public/ts/area-landing.ts

```diff
@@ -1,7 +1,23 @@
 // Sass
 import "../sass/index.scss";
 // Ts imports
-import { MapComponent } from "./components/map";
+import("mapbox-gl");
+import { IMapComponentOptions, IMapPadding, MapComponent} from "dp-maps-js";
+import "../json/geoData1.json"; // TODO test
```

Comment:
This is to test that we can serve static json files from the web app... can be removed as it's just for debugging...